src/scripts/start-batch-main.js

```diff
@@ -3,9 +3,15 @@ import * as recipe from "./ui-elements/recipe.js"
 
 util.loadJS("./scripts/ui-elements/properties.js");
 
-new recipe.recipe('pilsner', 600, 435);
-new recipe.recipe('wheat', 300, 157);
-new recipe.recipe('ipa', 150, 94);
-new recipe.recipe('stout', 200, 200);
-new recipe.recipe('ale', 100, 92);
-new recipe.recipe('alcoholFree', 125, 95, 'alcohol free');
+fetch('https://api.bierproductie.nymann.dev/recipes/')
+.then(response => response.json())
+.then(data => {
+    console.log(data);
+    data.forEach(r => {
+        if(r.name === "alcoholFree"){
```

Comment:
Why check if it is the alcohol free recipe?

Reply:
When we get the data from API  it is called "alcoholFree", 
so we want to make it say "alcohold Free" in the html instead right?

Reply:
We can remove it, when the html will just show alcoholFree

Reply:
Aight, makes sense.